.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,65 @@
+name: Feature request
+description: Suggest a new feature for MSAL .NET.
+labels: [feature request]
+title : '[Feature Request] '
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before Submitting your Feature Request
+      Please make sure that your question or issue is not already covered in [MSAL docs](https://learn.microsoft.com/en-us/entra/msal/dotnet/) or [samples](https://learn.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code?tabs=apptype).
+      
+- type: markdown
+  attributes:
+    value: |
+      ## Feature Request
+      
+- type: dropdown
+  attributes:
+    label: Public, Confidential or Managed Identity Client?
+    description: Are you using PublicClientApplication (desktop and mobile apps), ConfidentialClientApplication (web sites, web apis, S2S) or ManagedIdentityApplication?
+    multiple: true
+    options: 
+      - "Public"
+      - "Confidential"
+      - "Managed Identity"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Description
+    description: "Is your feature request related to a problem? Please describe."
+  validations: 
+    required: true
+
+- type: textarea
+  attributes:
+    label: Solution
```

Comment:
I'd recommend removing this altogether - we should not be asking our customers for specific implementation details or feature solutions 😀 That's up to the engineering and the PM team to determine based on _the customer problem_.

Reply:
Some customers do have solutions though, and propose PRs to fix them.

Reply:
This is also not necessarily related to implementation details (although some folks do provide this), but could be something more general like - "I want MSAL to have a `AcquireTokenWithBlah` method that accepts `Blah` and does a thing.".

Reply:
Maybe then this section should be renamed to "Proposal" since then it is framed more as a suggestion rather than a solution that the team needs to implement.

Reply:
I like the word "Proposal" better. We are already asking for this in the current template and customers are providing high level proposals like @pmaytak suggested. If they want to provide implementation details, they will usually open a PR.

Reply:
Yea, I think currently it's 'Proposed solution'. I think it'd be good to keep it, make it optional, good to hear users' opinions/perspectives.